src/connection/tests.rs

```diff
@@ -794,14 +807,18 @@ fn from_connect_fetchall_rollback_panic_closed() {
 
     connection.status = ConnectionStatus::Closed;
     match connection.rollback() {
-        Ok(_x) => {}
-        Err(err) => panic!("Fetching failed: {}", err),
+        Ok(_) => {
+            panic!("Rollback should not pass");
+        }
+        Err(err) => {
+            assert!(format!("{}", err).contains("is closed"));
+        }
```

Comment:
Instead of this
```rust
    match connection.rollback() {
        Ok(_) => {
            panic!("Rollback should not pass");
        }
        Err(err) => {
            assert!(format!("{}", err).contains("is closed"));
        }
    }
```
I would write:
```rust
    let rollback_res = connection.rollback();
    assert!(rollback_res.is_err());
    assert!(format!("{}", rollback_res.err()).contains("is closed"));
    
```